### ISSUE TITLE
Upgrade path_provider

### ISSUE THREAD

**Issue by Panthro** [luanpotter/flame#69]
```Because flame 0.10.3 depends on path_provider ^0.4.1 and no versions of flame match >0.10.3 <0.11.0, flame ^0.10.3 requires path_provider ^0.4.1.
So, because xxx depends on both flame ^0.10.3 and path_provider ^0.5.0, version solving failed.
pub get failed (1)
```

**Comment by Panthro** [comment]
just a reminder to self (or anyone else) to create a PR for this.

**Comment by erickzanardo** [comment]
Fear not @Panthro , this update will be happening on this PR: https://github.com/luanpotter/flame/pull/68/files#diff-fb2d52cd953afedcafc9fe7629c6f132R15

**Comment by luanpotter** [comment]
Are we ready to merge that, though, @erickzanardo ? Or should we split the PR? The whole thing LGTM, AFAIC. But it's a breaking change, ofc. What do you think?

**Comment by erickzanardo** [comment]
@luanpotter right now, no. But I may be able to finish that PR tomorrow.

**Comment by luanpotter** [comment]
Ok, that's perfectly reasonable. We release a new version tomorrow-ish then.

**Comment by erickzanardo** [comment]
Resolvd on #68

**Comment by luanpotter** [comment]
Released on 0.11.0
